src/client/components/ReviewAndSubmitSection.jsx

```diff
@@ -45,28 +45,28 @@ class ReviewAndSubmitSection extends React.Component {
 
         {/* TODO(crew): Change names of sections to real names. */}
         <ReviewCollapsiblePanel
-            sectionLabel="NameAndGeneralInfoSection"
-            updatePath="/personal-information/name-and-general-information"
-            component={<NameAndGeneralInfoSection reviewSection/>}/>
+            sectionLabel="PersonalInfoSection"
+            updatePath="/veteran-information/name-and-general-information"
+            component={<PersonalInfoSection reviewSection/>}/>
+
+        <ReviewCollapsiblePanel
+            sectionLabel="DemographicInformationSection"
+            updatePath="/veteran-information/demographic-information"
+            component={<DemographicInformationSection reviewSection/>}/>
 
         <ReviewCollapsiblePanel
             sectionLabel="VAInformationSection"
-            updatePath="/personal-information/va-information"
+            updatePath="/veteran-information/demographic-information"
             component={<VAInformationSection reviewSection/>}/>
 
         <ReviewCollapsiblePanel
             sectionLabel="AdditionalInformationSection"
-            updatePath="/personal-information/additional-information"
+            updatePath="/insurance-information/additional-information"
```

Comment:
I didn't really touch review panel in this PR because their were other things that need to get cleaned up so I thought that would be the next PR. I can add it to this PR though.